### ISSUE TITLE
`entmax_bisect` is not stable around `alpha=1`

### ISSUE THREAD

**Issue by MartinXPN** [deep-spin/entmax#24]
First of all, thanks for a great library! It's very nicely implemented!

From the documentation provided in the code, I understood that `entmax_bisect` should behave like `softmax` when `alpha` is set to 1.

I've done some experiments and the results seem to be different from softmax when the alpha is equal to 1. Yet, when it's close to 1 it approximates the softmax behavior.

Here is the code snippet for a very small example:
```python
import torch
from entmax import entmax_bisect


torch.softmax(torch.Tensor([0., 0., 1.]), dim=-1)                        # tensor([0.2119, 0.2119, 0.5761])

entmax_bisect(torch.Tensor([0., 0., 1.]), dim=-1, alpha=0.9)             # tensor([0.2195, 0.2195, 0.5611])
entmax_bisect(torch.Tensor([0., 0., 1.]), dim=-1, alpha=0.95)            # tensor([0.2157, 0.2157, 0.5687])
entmax_bisect(torch.Tensor([0., 0., 1.]), dim=-1, alpha=0.99)            # tensor([0.2127, 0.2127, 0.5747])
entmax_bisect(torch.Tensor([0., 0., 1.]), dim=-1, alpha=0.999999)        # tensor([0.2119, 0.2119, 0.5761])
entmax_bisect(torch.Tensor([0., 0., 1.]), dim=-1, alpha=1)               # tensor([0.3333, 0.3333, 0.3333]) <--
entmax_bisect(torch.Tensor([0., 0., 1.]), dim=-1, alpha=1.00001)         # tensor([0.2119, 0.2119, 0.5761])
entmax_bisect(torch.Tensor([0., 0., 1.]), dim=-1, alpha=1.1)             # tensor([0.1985, 0.1985, 0.6031])
```

Is this a bug or is it the intended behavior?
I think it's not very clear from the documentation.

**Comment by vene** [comment]
see https://github.com/deep-spin/entmax/issues/23#issuecomment-801012154

the bisection algorithm is expected not to work with alpha=1, but also, it makes little sense to use it, because the regular torch.nn.softmax would be faster and better for this case.

we will clarify the documentation and then close these issues if that's alright

**Comment by MartinXPN** [comment]
Sure! sounds good

**Comment by vene** [comment]
docstrings updated, thanks for reporting
